pkg/descheduler/pod/pods.go

```diff
@@ -234,3 +236,28 @@ func SortPodsBasedOnAge(pods []*v1.Pod) {
 		return pods[i].CreationTimestamp.Before(&pods[j].CreationTimestamp)
 	})
 }
+
+// get number of replication pods from deployment or job
+func GetReplicasCount(podLister listersv1.PodLister) map[string]uint {
+	podReplicasCount := make(map[string]uint)
+
+	// pods, err := podInformer.Lister().List(labels.Everything())
+	pods, err := podLister.List(labels.Everything())
+	if err != nil {
+	}
+
+	for _, pod := range pods {
+
+		ownerRefList := OwnerRef(pod)
+		if len(ownerRefList) == 0 {
+			continue
+		}
+
+		for _, ownerRef := range ownerRefList {
+			s := strings.Join([]string{pod.ObjectMeta.Namespace, ownerRef.Kind, ownerRef.Name}, "/")
+			podReplicasCount[s] = podReplicasCount[s] + 1
```

Comment:
What does a replica count means in this context? Based on the code `podReplicasCount` means a number of owner references instead of number of replicas a RS, RC, SS, ... is expected to have. Would you please elaborate more on the meaning?